.github/workflows/update-docs.yml

```diff
@@ -0,0 +1,36 @@
+name: PyPI
```

Comment:
I would just call the file `docs.yml`, without a verb.

Reply:
Renamed to `gh-pages.yml`, which makes more sense since im already auto-deploying to readthedocs (https://goosebit.rtfd.io)